app/src/main/java/net/sylvek/itracing2/receivers/CustomAction.java

```diff
@@ -31,6 +31,10 @@ public void onReceive(Context context, Intent intent)
         if (action.startsWith("http://")) {
             new CallUrl<HttpsURLConnection>(action).start();
         }
+
+        if (!action.isEmpty()) {
```

Comment:
not sure you need that complexity now, some actions string could be long  and user will have to add this prefix..